test/e2e/libpod_suite_varlink_test.go

```diff
@@ -1,4 +1,4 @@
-// +build remoteclientvarlink
```

Comment:
Did you mean to get these tags too?

Reply:
No, but I don't see where this tag ever gets set.